# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../busybox/declarations.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- busybox-1.3.0/libbb/bb_strtonum.c.orig	2006-12-29 02:43:13.000000000 -0800
+++ busybox-1.3.0/libbb/bb_strtonum.c	2006-12-29 02:46:41.000000000 -0800
@@ -98,7 +98,7 @@
 #endif
 
 #if UINT_MAX != ULONG_MAX
-unsigned bb_strtou(const char *arg, char **endp, int base)
+unsigned long bb_strtou(const char *arg, char **endp, int base)
 {
 	unsigned long v;
 	char *endptr;
@@ -110,7 +110,7 @@
 	return handle_errors(v, endp, endptr);
 }
 
-int bb_strtoi(const char *arg, char **endp, int base)
+long int bb_strtoi(const char *arg, char **endp, int base)
 {
 	long v;
 	char *endptr;
